Documentation/GettingStarted.md

```diff
@@ -1062,24 +1062,25 @@ Request determines is it a GET request, or a POST request, what is the request b
 This is how you can create a simple GET request
 
 ```swift
-let request = NSURLRequest(URL: NSURL(string: "http://en.wikipedia.org/w/api.php?action=parse&page=Pizza&format=json")!)
+let req = NSURLRequest(url: NSURL(string: "http://en.wikipedia.org/w/api.php?action=parse&page=Pizza&format=json") as! URL)
```

Comment:
Yeah, this will need to be changed ... Good catch though. I'll just change this immediately after merge.